tests/telemetry/test_writer.py

```diff
@@ -137,63 +113,41 @@ def test_app_started_event_configuration_override(test_agent_session, ddtrace_ru
 
     env = os.environ.copy()
     # Change configuration default values
+    env["DD_TRACE_ENABLED"] = "false"
     env["DD_DATA_STREAMS_ENABLED"] = "true"
+    env["DD_APPSEC_ENABLED"] = "true"
+    env["DD_PROFILING_ENABLED"] = "true"
+    env["DD_DYNAMIC_INSTRUMENTATION_ENABLED"] = "true"
     env["DD_TRACE_PROPAGATION_STYLE_EXTRACT"] = "b3multi"
     env["DD_TRACE_PROPAGATION_STYLE_INJECT"] = "datadog"
     env["DD_TRACE_OTEL_ENABLED"] = "true"
     env["DD_RUNTIME_METRICS_ENABLED"] = "true"
+    env["DD_EXCEPTION_DEBUGGING_ENABLED"] = "true"
+    if PY2:
+        # Prevents gevent importerror when profiling/debugger are enabled
+        env["DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE"] = "false"
```

Comment:
@P403n1x87 Here's a reproduction:

Command: `pip install ddtrace==1.16.0 && DD_PROFILING_ENABLED=true python2.7 -c "import logging; logging.basicConfig(); import ddtrace.auto"`

Output
```
WARNING:ddtrace.bootstrap.sitecustomize:error configuring Datadog tracing
Traceback (most recent call last):
  File "/Users/munirabdinur/go/src/github.com/DataDog/dd-trace-py/ddtrace/bootstrap/sitecustomize.py", line 194, in <module>
    cleanup_loaded_modules()
  File "/Users/munirabdinur/go/src/github.com/DataDog/dd-trace-py/ddtrace/bootstrap/sitecustomize.py", line 93, in cleanup_loaded_modules
    do_cleanup = any(is_module_installed(m) for m in MODULES_REQUIRING_CLEANUP)
  File "/Users/munirabdinur/go/src/github.com/DataDog/dd-trace-py/ddtrace/bootstrap/sitecustomize.py", line 93, in <genexpr>
    do_cleanup = any(is_module_installed(m) for m in MODULES_REQUIRING_CLEANUP)
  File "/Users/munirabdinur/go/src/github.com/DataDog/dd-trace-py/ddtrace/bootstrap/sitecustomize.py", line 78, in is_module_installed
    return find_loader(module_name) is not None
  File "/Users/munirabdinur/.pyenv/versions/2.7.18/lib/python2.7/pkgutil.py", line 473, in find_loader
    loader = importer.find_module(fullname)
  File "ddtrace/vendor/wrapt/decorators.py", line 470, in _synchronized
    return wrapped(*args, **kwargs)
  File "ddtrace/vendor/wrapt/importer.py", line 217, in find_module
    __import__(fullname)
ImportError: No module named gevent
```

Reply:
Ah weird, `find_loader` is trying to import `gevent` here, even though it's a root module